fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/FHIRDbDAO.java

```diff
@@ -21,32 +21,18 @@
     public static final String PROPERTY_DB2_PSWD = "password";
```

Comment:
Why have the constants if these still exist?

Reply:
should be refactored...although this is now probably dead code, so some cleanup is required. Will create a separate issue.

Reply:
issue #1305 to track.